### PR TITLE
Update summary.

### PR DESCRIPTION
**Pull request by Chasbob**
Clean titles to be a bit more Alexa friendly.

### CODE REVIEW

File: bhamcal/extractor.py [hunk 1]
```diff
@@ -69,3 +69,20 @@ def extract_datetime(date, time):
     dt = DEFAULT_TIMEZONE.localize(dt)
     dt = dt.astimezone(pytz.utc)
     return dt
+
+
+def clean_subject(subject: str) -> str:
+    # Was not needed in my case but I'll assume others do.
+    one = r"^LI "
+    # Remove module codes, LM and/or LH and extended.
+    two = r"(?P<code>\([0-9]+/[0-9]+\))|(?P<prefix>LM/LH|LH/LM|LH|LM)|(?P<extended>\(Extended\)?)"
```
Comment (jedevc) [inline bhamcal/extractor.py]: LI is another prefix, like LH or LM. So they should probably be handled similarly?

File: bhamcal/extractor.py [hunk 2]
```diff
@@ -69,3 +69,20 @@ def extract_datetime(date, time):
...
+    # Remove duplicates in the case of the name being present twice on some extended modules.
+    three = r"(?P<one>^.*/)"
+    round_one_re = re.compile(one,re.MULTILINE)
```
Comment (jedevc) [inline bhamcal/extractor.py]: format nit around comma

File: bhamcal/extractor.py [hunk 3]
```diff
@@ -69,3 +69,20 @@ def extract_datetime(date, time):
...
+    round_two_re = re.compile(two, re.MULTILINE)
+    round_three_re = re.compile(three, re.MULTILINE)
+    round_one = re.sub(round_one_re,"",subject)
```
Comment (jedevc) [inline bhamcal/extractor.py]: format nit

File: bhamcal/extractor.py [hunk 4]
```diff
@@ -69,3 +69,20 @@ def extract_datetime(date, time):
...
+
+def clean_subject(subject: str) -> str:
+    # Was not needed in my case but I'll assume others do.
+    one = r"^LI "
+    # Remove module codes, LM and/or LH and extended.
+    two = r"(?P<code>\([0-9]+/[0-9]+\))|(?P<prefix>LM/LH|LH/LM|LH|LM)|(?P<extended>\(Extended\)?)"
+    # Remove duplicates in the case of the name being present twice on some extended modules.
+    three = r"(?P<one>^.*/)"
+    round_one_re = re.compile(one,re.MULTILINE)
+    round_two_re = re.compile(two, re.MULTILINE)
```
Comment (jedevc) [inline bhamcal/extractor.py]: nit: compile these outside of the function

Comment (Chasbob) [reply bhamcal/extractor.py]: Cheers forgot about that.

**Pull request merged**